string text
uint32 uid
uint8 answer_type
